docker-start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+[ -z "$UID" ] && UID=0
+[ -z "$GID" ] && GID=0
+
+# echo >> /etc/xxx and not adduser/addgroup because adduser/addgroup won't work if uid/gid already exists
+echo -e "xuser:x:${UID}:${GID}:xuser:/home/xuser:/bin/ash\n" >> /etc/passwd
```

Comment:
Yes, `/bin/ash` isn't necessary (used it to tune the script interactively, but not needed in the final container)
And yes, /app is better even if I don't see where it would be needed
I'll update uidgid2 accordingly